### ISSUE TITLE
New Integration Request - Azure Devops to Collection

### ISSUE THREAD

**Issue by minoseah629** [a85/POSTMan-Chrome-Extension#12858]
### Describe your current workflow

I have 30+ microservice git repos in azure devops.  
Each microservice has a respective E2E collection .
Current process is team member generates a fork off of the main team's workspace.
Fork is placed in a sub team's workspace. 
Team member makes modifications to E2E collection. 
Team member generates Pull Request and approval process occurs.

### Describe your ideal workflow

When team member is working within a particular microservice git repo, the E2E collection is inside the git repo. 
So the team member can generate a pull request within azure devops for the E2E collection instead of directly inside postman.  

### Describe alternatives you've considered

Basically, asking for functionality of Backup a Collection to Github in Azure Devops

### Additional context

I know Postman has existing Azure Devops integration between API. This integration does not meet my needs.

**Comment by symbiunt** [comment]
Any updates on when this might be implemented?

**Comment by shashankawasthi88** [comment]
@minoseah629 @symbiunt we are actively working on connecting Collections to git repos. Can we get onto a call to discuss your workflows in detail? We have a proposed solution and we want to ensure it works for your usecase. 

You can schedule a call with us using this [link](https://calendly.com/shashank-awasthi/specification-designer-clone). I am the product manager working on this initiative.

**Comment by shashankawasthi88** [comment]
Hi @minoseah629 @symbiunt ,

As of [Postman v12](https://www.postman.com/downloads/), we now support Native Git Integration.
You can connect a workspace directly to a local Git repository and work against your filesystem. Collections and related artifacts are stored as files, enabling standard Git workflows (branching, diffs, PRs, CI/CD) without manual export/import. This will allow you to commit and push changes to your Azure DevOps repositories.
Docs: https://learning.postman.com/docs/agent-mode/native-git
Announcement: https://blog.postman.com/new-postman-is-here/
Feel free to try it out and share feedback.

https://github.com/user-attachments/assets/121148e8-f831-4713-ad76-4fa5c5f16593
